Python-Home-Challenges/Yarin_solution.py

```diff
@@ -0,0 +1,21 @@
+
+def most_common_word(file_name):
```

Comment:
+1 for using functions
but, best practice, is to create few functions, with each one of them doing 1 thing only